ci/310-oldest.yaml

```diff
@@ -0,0 +1,59 @@
+name: test
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - six
+  - pyyaml
+  - deprecated
+  # required
+  # testing
+  - matplotlib
+  - pytest
+  - pytest-cov
+  - coverage
+  - codecov
+  # optional
+  - bokeh
+  - scikit-learn
+  - geopandas=0.12
+  - quantecon
+  - rtree
+  - tqdm
+  - quilt3
+  - rasterio
+  - rasterstats
+  - pytest
+  - coverage
+  - coveralls
+  - ipywidgets
+  - twine
+  - urbanaccess
+  - urllib3>=1.26
+  - wheel
+  - dill
+  - pytest
+  - pytest-cov
+  - codecov
+  - bokeh
+  - libpysal
+  - esda
+  - spaghetti
+  - tobler>=0.2.1
+  - spint
+  - spreg
+  - spvcm
+  - mgwr
+  - segregation
+  - pointpats
+  - giddy
+  - mapclassify
+  - seaborn
+  - descartes
+  - numba
+  - momepy
+  - inequality
+  - access
+  - splot
+  - pip:
```

Comment:
Shall we move `spopt` into the `dependencies` section (in all environments)?

Reply:
Gentle ping to remind to remove the `pip: spopt` and move `spopt` directly up beneath `splot` in an CI environments for this PR.